ch3/download_model_fluent.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+import mlflow
+import pandas as pd
+import psycopg2
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+
+# 0. set mlflow environments
+MLFLOW_TRACKING_URI = "http://localhost:5000"
+MLFLOW_S3_ENDPOINT_URL = "http://localhost:9000"
+
+os.environ["MLFLOW_S3_ENDPOINT_URL"] = MLFLOW_S3_ENDPOINT_URL
+os.environ["MLFLOW_TRACKING_URI"] = MLFLOW_TRACKING_URI
+os.environ["AWS_ACCESS_KEY_ID"] = "mystorage"
+os.environ["AWS_SECRET_ACCESS_KEY"] = "mystoragepw"
+
+# 1. get data
+db_connect = psycopg2.connect(
+    host="localhost",
+    database="postgres",
+    user="postgres",
+    password="mypassword",
+)
```

Comment:
```suggestion
db_connect = psycopg2.connect(
    user="myuser",
    password="mypassword",
    host="localhost",
    port=5432,
    database="mydatabase",
)
```
user, password, host, port, database 순서대로 적어주면 좋을 것 같습니다.
그리고 port가 default로 5432이긴하나, port를 명시적으로 기입하면 좋을 것 같습니다.

Reply:
반영되었습니다 :)